cmd/tools/db-seeder/main.go

```diff
@@ -0,0 +1,205 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"path"
+	"sync"
+	"time"
+
+	"github.com/jackc/pgx/v5/pgxpool"
+	"sc-bos-db-seeder/pkg/seeder/air_quality"
+	"sc-bos-db-seeder/pkg/seeder/meter"
+	"sc-bos-db-seeder/pkg/seeder/occupancy"
+
+	"github.com/smart-core-os/sc-golang/pkg/trait"
+	"github.com/vanti-dev/sc-bos/pkg/app/appconf"
+	"github.com/vanti-dev/sc-bos/pkg/app/sysconf"
+	"github.com/vanti-dev/sc-bos/pkg/driver/alldrivers"
+	mockConfig "github.com/vanti-dev/sc-bos/pkg/driver/mock/config"
+	airqualityConfig "github.com/vanti-dev/sc-bos/pkg/zone/feature/airquality/config"
+	occupancyConfig "github.com/vanti-dev/sc-bos/pkg/zone/feature/occupancy/config"
+
+	"github.com/vanti-dev/sc-bos/pkg/zone/allzones"
+	metersConfig "github.com/vanti-dev/sc-bos/pkg/zone/feature/meter/config"
```

Comment:
Convention is that package (or import) names should be lowercase, ideally single words (but two is OK) without underscores. https://go.dev/doc/effective_go#package-names